swan-lake/integration-tools/openapi-tool.md

```diff
@@ -942,3 +942,201 @@ The attributes of the annotation are optional and can be used for each particula
 | `licenseName: string`       | You can use this to add the name of the license under which the API is provided.                                                                                                                                                                                                                                                          | Optional          |
 | `licenseURL: string`        | You can use this to add the URL details regarding the full text of the license.                                                                                                                                                                                                                                                           | Optional          |
 | `embed: string`             | Turns on generating OpenAPI documentation for the service for [introspection endpoint](https://ballerina.io/spec/http/#236-openapi-specification-resources) support when used with `true` in the annotation.                                                                                                                                                                                               | Optional          |
+
+## OpenAPI Contract Modifier
+
+### Modify the OpenAPI Contract Using Ballerina-Friendly Naming Conventions
+
+The `sanitize` subcommand sanitizes the OpenAPI contract file according to the naming conventions of Ballerina. The Ballerina-specific name extension `x-ballerina-name` is added to the schemas, including query names and object property names, which cannot be modified directly.
+
+```
+bal openapi sanitize [-i | --input] <openapi-contract-file-path>
+                     [-o | --output] <output-file-path>
+                     [-n | --name] <generated-file-name>
+                     [-f | --format] [json|yaml]
+                     [-t | --tags] <tag-names>
+                     [--operations] <operation-names>
+```
+The command-line arguments below can be used with the command for each particular purpose as described below.
+
+| Command option            | Description                                                                                                                                                                                                               | Mandatory/Optional |
+|---------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|
+| `-i \| --input`           | The `openapi-contract-path` command option specifies the path of the OpenAPI contract file (e.g., `my-api.yaml` or `my-api.json`).                                                                                        | Mandatory          |
```

Comment:
Where is `openapi-contract-path` coming from?

Reply:
Fixed via [0ffae86](https://github.com/ballerina-platform/ballerina-dev-website/pull/9626/commits/0ffae869cfe37038505105475a276d0c539f066c)